vcpkg/users/platforms/uwp.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Universal Windows Platform
+description: Building for UWP with the Microsoft Visual C++ Compiler
+ms.date: 04/28/2023
+---
+# Universal Windows Platform (UWP)
+
+## Triplets
+
+vcpkg includes [triplets](https://github.com/microsoft/vcpkg/tree/master/triplets) for building UWP applications using the MSVC ``cl.exe`` compiler.
+
+| Architecture | vcpkg triplets |
+|--------------|----------------|
+| x64          | x64-uwp        |
+| arm          | arm-uwp        |
+
+There are also additional [community triplets](https://github.com/microsoft/vcpkg/tree/master/triplets/community) for UWP as well.
+
+| Architecture | vcpkg community triplets |
+|--------------|--------------------------|
+| x86          | x86-uwp                  | 
+| arm64        | arm64-uwp                |
+
+> [!NOTE]
+> Community triplets are not tested as part of vcpkg repository's CI process, so regressions can occur as part of library updates. PRs improving support are welcome!
```

Comment:
We should not copy the definition of community triplets into every page that mentions them.

```suggestion
```

Reply:
Fixed.